### DISCUSSION TITLE
Do you have an official logo ?

### DISCUSSION THREAD

**Discussion by Yann39** [sablierapp/sablier#206] [General]
Hi,
Just wondering if you have an official logo.
i.e. I list the tools I use in a dashboard and I'm missing a logo for Sablier 😁

![image](https://github.com/acouvreur/sablier/assets/1263086/c0e0eadb-f1ee-4395-a817-9db7be6887e4)

**Comment by acouvreur** [comment]
Hey! I don't have an official logo, but I've used one of these ones: https://www.flaticon.com/free-icons/hourglass

**Reply by TheRedCyclops** [reply]
then it would be a good idea to add attribution to prevent any trouble

**Reply by acouvreur** [reply]
It's only used on the website where it's credited on the main page:
![image](https://github.com/user-attachments/assets/b005c32d-97bd-4795-a307-6acf5fba9299)

Should I credit it somewhere else ?

**Reply by Yann39** [reply]
I think it would be nice to have your own logo, like many other open source tools easily identifiable by their logo.
I can do some propositions if you want (with no credit required).

For example I was thinking of something like this representing containers going live or down :

![sablier_test](https://github.com/user-attachments/assets/19bb8627-9a62-4785-8140-2aeedb177ca8)

It is AI generated but can serve as a base to an hand-crafted SVG version.

Of course if you don't like it, or were thinking of something else, or simply don't want a logo, I won't insist :)

**Reply by acouvreur** [reply]
Yeah, I've always wanted to create a logo myself with a gopher holding some kind of hourglass. Haha, feel free to provide some logo that you like!

**Reply by acouvreur** [reply]
Now we have official artwork https://github.com/sablierapp/artwork

**Comment by acouvreur** [comment]
Official artwork now here: https://github.com/sablierapp/artwork
